tests/bud.bats

```diff
@@ -6624,3 +6624,26 @@ _EOF
   expect_output --substring "$podman_files"
   expect_output --substring "$podman_processes"
 }
+
+@test "build no write file on host - CVE-2024-1753" {
+  _prefetch alpine
+  cat > ${TEST_SCRATCH_DIR}/Containerfile << _EOF
+FROM alpine as base
+
+RUN ln -s / /rootdir
+
+FROM alpine
+
+RUN echo "With exploit show host root, not the container's root, and create /BIND_BREAKOUT in / on the host"
```

Comment:
I won't block over it but this really seem like unnecessary line, a extra RUN instruction is just slowing down the test for no reason. Sure it is basically nothing in the grand scheme but every single extra step that we do not need is making things slower than they have to be.